src/index.ts

```diff
@@ -555,6 +560,7 @@ prog
       'ignore-pattern': string;
       'write-file': boolean;
       'report-file': string;
+      'max-warnings': number | undefined;
```

Comment:
```suggestion
      'max-warnings': Infinity;
```

Seems like an accurate default to me

Reply:
ah I see you can't use default args in an interface like this, sorry for the incorrect recommendation